Parsers/ASimNetworkSession/ProductParsers/ASimNetworkSessionPaloAltoPanOS.yaml

```diff
@@ -0,0 +1,118 @@
+Parser:
+  Title: Palo Alto Network Sessions
+  Version: '0.1'
+  LastUpdated: Dec 20, 2021
+Product:
+  Name: PaloAlto
+Normalization:
+  Schema: NetworkSessions
+  Version: '0.2.0'
```

Comment:
Current Version is 0.2.1. Also check the [schema updates](https://docs.microsoft.com/en-us/azure/sentinel/network-normalization-schema#schema-updates) to see if you need to update the parser based on them.